### PR TITLE
feat: message documentation

### PR DESCRIPTION
**Pull request by houbly**
- Documentation converted to markdown
- Updated Example.njk - for wmndsMsgSummary and wmndsMsgHelp
- Updated Nunjucks components params - for wmndsMsgSummary and wmndsMsgHelp across all files
- Added properties for the new component params - for wmndsMsgSummary and wmndsMsgHelp

### CODE REVIEW

File: src/wmnds/components/message/summary/_properties.njk.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "wmndsMsgSummaryProps": [
+    {
+      "name": "type",
+      "type": "string",
+      "description": "Type of the message component. As default it is <code class='wmnds-website-inline-code'>info</code>. Available types: <code class='wmnds-website-inline-code'>info</code>, <code class='wmnds-website-inline-code'>success</code>, <code class='wmnds-website-inline-code'>success-fill-no-icon</code>, <code class='wmnds-website-inline-code'>warning</code>, <code class='wmnds-website-inline-code'>error</code>"
+    },
+    {
+      "name": "title",
+      "type": "string",
+      "description": "Type of the button. As default it is <code class='wmnds-website-inline-code'>null</code>. Available types: <code class='wmnds-website-inline-code'>link</code>, <code class='wmnds-website-inline-code'>primary</code>, <code class='wmnds-website-inline-code'>secondary</code>, <code class='wmnds-website-inline-code'>destructive</code>, <code class='wmnds-website-inline-code'>start</code>, <code class='wmnds-website-inline-code'>mode</code> "
```
Comment (catiarodriguescosta) [inline src/wmnds/components/message/summary/_properties.njk.json:11]: Wrong description

File: src/wmnds/components/message/summary/_properties.njk.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "subtitle",
+      "type": "string",
+      "description": "Type of the button. As default it is <code class='wmnds-website-inline-code'>null</code>. Available types: <code class='wmnds-website-inline-code'>link</code>, <code class='wmnds-website-inline-code'>primary</code>, <code class='wmnds-website-inline-code'>secondary</code>, <code class='wmnds-website-inline-code'>destructive</code>, <code class='wmnds-website-inline-code'>start</code>, <code class='wmnds-website-inline-code'>mode</code> "
```
Comment (catiarodriguescosta) [inline src/wmnds/components/message/summary/_properties.njk.json:16]: Wrong description

**Review by catiarodriguescosta — CHANGES_REQUESTED** [review]
Left a few notes! 👍

File: src/wmnds/patterns/content-cards/travel-updates/_travel-updates.njk [hunk 3]
```diff
@@ -51,7 +51,7 @@
         <!-- Help message -->
         {{
             wmndsMsgHelp({
-                message: 'Save bus routes to your homepage by pressing the star icon'
+                contentText: 'Save bus routes to your homepage by pressing the star icon'
             }) | indent(12) | trim
```
Comment (catiarodriguescosta) [inline src/wmnds/patterns/content-cards/travel-updates/_travel-updates.njk:55]:
```suggestion
            })
```

File: src/wmnds/patterns/question/confirmation-message/_confirmation-message.njk [hunk 4]
```diff
@@ -7,7 +7,7 @@
         type: 'success-fill-no-icon',
         title: 'Request complete',
         subtitle: 'You can now start using your Swift card',
-        message: 'Your reference number is <b>12345</b>',
+        contentHTML: 'Your reference number is <b>12345</b>',
         classes: 'wmnds-text-align-center'
     }) | trim
```
Comment (catiarodriguescosta) [inline src/wmnds/patterns/question/confirmation-message/_confirmation-message.njk:12]:
```suggestion
    })
```

**Review by catiarodriguescosta — APPROVED** [review]
Just found those two tiny indentation things. Will approve it anyway! 👍

**Review by daylesalmon — CHANGES_REQUESTED** [review]
Just a few changes.

**Review by daylesalmon — APPROVED** [review]

**Pull request merged**